src/libraries/Microsoft.PowerFx.Core/Texl/Intellisense/MarkdownString.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.PowerFx.Intellisense
+{
+    /// <summary>
+    /// A string that represents Github flavored markdown. 
+    /// For use in displaying in intellisense. 
+    /// </summary>
+    public class MarkdownString
```

Comment:
>MarkdownString [](http://example.com/codeflow?start=17&length=14)

introducing a wrapper so that the type system can help us with "plain text" vs. "markdown".